src/Validators/IntegrityChecks/Elements/DateValidationsCheck.cs

```diff
@@ -11,6 +12,8 @@ public IntegrityCheckResult Validate(IElement element)
             if (element.Properties is null)
                 return result;
 
+            var relativeDateStringRegex = new Regex(@"^\d+-[dmy]-(ex|in)$");
```

Comment:
Would this fail the regex if we used say '5-D-EX' in the json? If so, do we want to force use of lowercase or allow upper or lower? One for discussion maybe

Reply:
If you adjust the Regex to (@"(?i)^\d+-[dmy]-(ex|in)$") it should set the option to ignore case (it's the (?i) that does this) and then you don't need the .ToLower later on